File: js/06/main.js
const array = require('../utilities/array');

const sumOfSquares = array.filledArray(100).reduce((acc, n) => acc + n * n, 0);
const squareOfSums = Math.pow(100 * 101 / 2, 2);
console.log(squareOfSums - sumOfSquares);

File: c/06/main.c
#include <stdio.h>
#include <math.h>

int main() {
  double sumOfSquares = 0;
  for (int i = 1; i <= 100; i++)
    sumOfSquares += pow(i, 2);

  double squareOfSums = pow(100 * 101 / 2, 2);

  printf("%f", squareOfSums - sumOfSquares);
}
